hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/DroolsRulesEngineImpl.java

```diff
@@ -182,6 +182,12 @@ public void fire() {
                     }
                 }
             }
+            long filteringTime = System.currentTimeMillis() - startFiltering;
+            log.debugf("Filtering BatchData [%s] took [%s]", batchData.size(), filteringTime);
+            if (filteringTime > PERF_FILTERING_THRESHOLD) {
+                log.warnf("Filtering Filtering BatchData [%s] took [%s] ms exceeding [%s] ms",
```

Comment:
I guess the deferring info is really a large amount of data, on perf scenarios, it's not really interesting I think.
We take the assumption that the ordering is working fine.
I'm going to move them to tracef just in case.
It's harmless and it will be there just if needed.